ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/CreateSupportTicketView.kt

```diff
@@ -0,0 +1,219 @@
+                localization = localization,
+            )
+
+            Spacer(modifier = Modifier.height(SECTION_TITLE_BOTTOM_PADDING))
+
+            DescriptionInputField(
+                description = description,
+                onDescriptionChange = { description = it },
+                enabled = !isSubmitting,
+                localization = localization,
+            )
+
+            Spacer(modifier = Modifier.height(SECTION_SPACING))
+
+            SubmitTicketButton(
+                email = email,
+                description = description,
+                isSubmitting = isSubmitting,
+                onSubmit = {
+                    isSubmitting = true
+                    hasError = false
+                    data.onSubmit(
+                        email,
+                        description,
+                        { /* Success - navigation handled by ViewModel */ },
+                        {
+                            isSubmitting = false
+                            hasError = true
+                        },
+                    )
+                },
+                localization = localization,
+            )
+        }
+
+        ErrorSnackbar(
+            hasError = hasError,
+            onErrorShow = { hasError = false },
+            localization = localization,
+            modifier = Modifier.align(Alignment.BottomCenter),
+        )
+    }
+}
+
+@Composable
+private fun EmailInputField(
+    email: String,
+    onEmailChange: (String) -> Unit,
+    enabled: Boolean,
+    localization: CustomerCenterConfigData.Localization,
+) {
+    OutlinedTextField(
+        value = email,
+        onValueChange = onEmailChange,
+        label = {
+            Text(
+                localization.commonLocalizedString(
+                    CustomerCenterConfigData.Localization.CommonLocalizedString.EMAIL,
+                ),
+            )
+        },
+        placeholder = {
+            Text(
+                localization.commonLocalizedString(
+                    CustomerCenterConfigData.Localization.CommonLocalizedString.ENTER_EMAIL,
+                ),
+            )
+        },
+        enabled = enabled,
+        modifier = Modifier
+            .fillMaxWidth()
+            .testTag("email_field"),
+        singleLine = true,
+        keyboardOptions = KeyboardOptions(
+            keyboardType = KeyboardType.Email,
+            imeAction = ImeAction.Next,
+        ),
+    )
+}
+
+@Composable
+private fun DescriptionInputField(
+    description: String,
+    onDescriptionChange: (String) -> Unit,
+    enabled: Boolean,
+    localization: CustomerCenterConfigData.Localization,
+) {
+    OutlinedTextField(
+        value = description,
+        onValueChange = onDescriptionChange,
+        label = {
+            Text(
+                localization.commonLocalizedString(
+                    CustomerCenterConfigData.Localization.CommonLocalizedString.DESCRIPTION,
+                ),
+            )
+        },
```

Comment:
should we set a max length? just a sanity thing. Does the backend prevent from sending huge chunks of text?